test/Microsoft.AspNet.Mvc.IntegrationTests/CollectionModelBinderIntegrationTest.cs

```diff
@@ -338,5 +339,204 @@ private class Person
             Assert.Equal(0, modelState.ErrorCount);
             Assert.True(modelState.IsValid);
         }
+
+        private class Person2
+        {
+            public int Id { get; set; }
+
+            [Required]
```

Comment:
These are dupes of tests that I wrote yesterday. This is supposed to be using `[BindRequired]` and friends. Sorry, I realize that wasn't clear in the mail.